ext/phashion_ext/extconf.rb

```diff
@@ -7,10 +7,22 @@
 $CFLAGS += " -fdeclspec" if RUBY_PLATFORM =~ /darwin/
 $includes = " -I#{HERE}/include"
 $libraries = " -L#{HERE}/lib -L/usr/local/lib"
+
+mac_os_with_homebrew = RUBY_PLATFORM =~ /darwin/ && system('which brew 2>&1 1>/dev/null')
+# Give include/lib paths for macOS using homebrew
+# https://github.com/westonplatter/phashion/pull/99
+if mac_os_with_homebrew
+  prefix = `brew --prefix`.strip
+  sqlite_prefix = `brew --prefix sqlite3`.strip
```

Comment:
sqlite3 in homebrew is "keg-only" meaning you can't find `sliqte3ext.h` at `/opt/homebrew/include`. Instead it only exists at ` /opt/homebrew/opt/sqlite/include`.